pandas/tests/extension/decimal/test_decimal.py

```diff
@@ -1,13 +1,19 @@
 import decimal
 
+import random
 import numpy as np
 import pandas as pd
 import pandas.util.testing as tm
 import pytest
 
 from pandas.tests.extension import base
 
-from .array import DecimalDtype, DecimalArray, make_data
+from .array import DecimalDtype, DecimalArray
+
+
+@pytest.fixture
+def make_data():
+    return [decimal.Decimal(random.random()) for _ in range(100)]
```

Comment:
What is the reason you moved this? There is already the `data` fixture

Reply:
these were previously in the ``array.py`` module itself, which is not a great idea.